pages/code-review.md

```diff
@@ -24,10 +24,34 @@ goals when reviewing code are to:
 - cultivate the ability to critique one’s code more strongly;
 - encourage open communication between the entire team.
 
+## Discuss code review with your team
+
+Every individual and team has a different set of expectations around code
+review. Discussing these expectations explicitly is the best way to make sure
+that everyone is on the same page.
+
+Bonus points for documenting and/or automating the outcomes of your discussion.
```

Comment:
so are you saying we should take out the "bonus points" part of the sentence above, @ccostino ? I agree with the sentiment of your thoughts, just not sure what you think the outcome should be. Perhaps we should suggest *where* a team might document this?